openrtx/include/core/graphics.h

```diff
@@ -223,8 +221,11 @@ uint8_t gfx_getFontHeight(fontSize_t size);
  * @param buf: char buffer
  * @return text width and height as point_t coordinates
  */
-point_t gfx_printBuffer(point_t start, fontSize_t size, textAlign_t alignment,
-                        color_t color, const char *buf);
+point_t gfx_printBuffer(point_t start,
```

Comment:
Probably is better to avoid having the pile of arguments, when they are more than three the linted version becomes a bit more messy than the original one